### PR TITLE
PoolingWidget/index - remove back btn on last step

### PR DESCRIPTION
**Pull request by W3stside**
Removes back button on last step (as per Anxos comment in #683)

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr702--dexreact.review.gnosisdev.com

File: src/components/PoolingWidget/LiquidityButtons.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const LiquidityButtons: React.FC<LiquidityButtonsProps> = ({
 }) => (
   <StepButtonsWrapper>
     {/* REMOVE BACK BUTTON ON TXRECEIPT */}
```
Comment (Velenir) [inline src/components/PoolingWidget/LiquidityButtons.tsx]: Also change this comment?

Comment (W3stside) [reply src/components/PoolingWidget/LiquidityButtons.tsx]: yesh

**Comment by anxolin** [comment]
@W3stside  I think this was merged too fast.

If I select 2 tokens, and then I want to change my selection, how do I do it now?
image

I wouldn't allow to go back only once the tx was sent

**Pull request merged**